src/common/src/config.rs

```diff
@@ -630,6 +630,18 @@ pub struct BatchConfig {
     /// Enable the spill out to disk feature for batch queries.
     #[serde(default = "default::batch::enable_spill")]
     pub enable_spill: bool,
+
+    /// Total maximum memory allowed for running message in the frontend.
+    #[serde(default = "default::batch::frontend_max_running_message_bytes")]
+    pub frontend_max_running_message_bytes: u64,
+
+    /// A message under this threshold will not be throttled for memory constraints.
+    #[serde(default = "default::batch::frontend_throttling_filter_min_bytes")]
+    pub frontend_throttling_filter_min_bytes: u64,
+
+    /// A message exceeding this threshold will always be throttled for memory constraints.
+    #[serde(default = "default::batch::frontend_throttling_filter_max_bytes")]
+    pub frontend_throttling_filter_max_bytes: u64,
```

Comment:
Why are they organized into the `batch` section? Shall we introduce a new section for frontend (pgwire) servers?